README-TEST.md

```diff
@@ -168,4 +168,32 @@ Files in ClusterTest
 * src/main/scala//net/snowflake/spark/snowflake/ClusterTestCheckResult.scala: The class to check test result
 * src/main/scala//net/snowflake/spark/snowflake/ClusterTest.scala: The driver class to run test
 * src/main/scala//net/snowflake/spark/snowflake/testsuite/ClusterTestSuiteBase.scala: Test case base class.
+* src/main/scala//net/snowflake/spark/snowflake/testsuite/LowMemoryStress.scala: Test upload large partition with available low memory.
 * Files in src/main/scala: the Scala test related files.
+
+## Specific test case description
+### Low memory test
+Test uploading large partition with a small amount of memory is defined in Cluster Test. The partition used in the test is of size 227 MB after compression and 330 MB before compression. By limiting the size of executor memory, multiple OOM errors can be reproduced.
+
+The connector hit OOM while trying to write data into a buffer. The processed size column indicates the processed data size before OOM.
+
+Before Mingli's multi-part upload was finished, OOM happens like the following:
```

Comment:
It's clearer to specific the Jira, instead the eng name.

Reply:
nice catch!